ads/google/doubleclick.js

```diff
@@ -55,18 +44,15 @@ export function doubleclick(global, data) {
 
   centerAd(global);
 
-  const gptFilename = selectGptExperiment(data);
-
-  writeAdScript(global, data, gptFilename);
+  writeAdScript(global, data);
```

Comment:
Maybe it makes sense to inline this function now. It is pretty small after this clean up.

Reply:
left it not inlined for ease of not rewriting too much testing, as all the tests of that file are testing that function.